.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!-- If this PR closes a JIRA ticket, make sure the title starts with the JIRA issue number, 
+for example JP-1234: <Fix a bug> -->
+Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
+
+<!-- If this PR closes a GitHub issue, reference it here by its number -->
+Closes #
+
+<!-- describe the changes comprising this PR here -->
+This PR addresses ...
+
+**Checklist**
+- [ ] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
+- [ ] updated relevant tests
+- [ ] updated relevant documentation
+- [ ] updated relevant milestone(s)
+- [ ] added relevant label(s)
```

Comment:
I copied the pull request template from `jwst` and modified it to explicitly ask to update `CHANGES.txt` in the relevant section